components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/index.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react'
+import * as qr from 'qr-image'
+import {
+  AccountSettingsNavTypes,
+  WalletAccountType,
+  UserAssetOptionType,
+  AssetOptionType
+} from '../../../../constants/types'
+import {
+  PopupModal,
+  TopTabNav,
+  AssetWatchlistItem
+} from '../..'
+import { AccountSettingsNavOptions } from '../../../../options/account-settings-nav-options'
+import { AssetOptions } from '../../../../options/asset-options'
+import { reduceAddress } from '../../../../utils/reduce-address'
+import { NavButton } from '../../../extension'
+import { Tooltip, SearchBar } from '../../../shared'
+import locale from '../../../../constants/locale'
```

Comment:
Small remark for a future PR - probably makes sense to use relative import paths.

Reply:
Sounds good, will need to do a refactor. Thank you!